.idea/compiler.xml

```diff
@@ -13,11 +11,17 @@
       <entry name="!?*.kt" />
       <entry name="!?*.clj" />
     </wildcardResourcePatterns>
-    <annotationProcessing>
-      <profile default="true" name="Default" enabled="false">
-        <processorPath useClasspath="true" />
-      </profile>
-    </annotationProcessing>
-    <bytecodeTargetLevel target="1.8" />
+    <bytecodeTargetLevel target="11">
```

Comment:
I thought you deleted .idea from git, is it necessary for anything?

Reply:
Thought so too, I must have lost the commit during a rebase, fixed